src/Cache.js

```diff
@@ -7,12 +7,31 @@ Logger.setLogLevel('ERROR')
 class Cache {
   constructor (store) {
     this._store = store
+    this.handlers = new Set();
```

Comment:
Can you explain this a bit? I don't see it used again

Reply:
Yes, it's a set of store addresses that is managed by `orbit-db` when creating/opening/closing caches. It helps keep track of which db is using which cache.

Reply:
Can some interface be added here to manipulate this property? It should be clear what its purpose is just from looking at this file

Reply:
Yeah, that's a good call. I can create an `addHandler` function that does a lot of what `OrbitDb._onClose` does